drltrace/drltrace_options.h

```diff
@@ -64,6 +64,12 @@ static droption_t<int> op_max_args
  "Maximum number of arguments to print.  This option allows to limit the number of "
  "arguments to be printed.  Specify 0 to disable args printing (including unknown).");
 
+static droption_t<std::string> op_config_file
+(DROPTION_SCOPE_CLIENT, "config", "./drltrace.config", "A path to drltrace's config file",
+ "Specify a path where config is located. The default value is \"./drltrace.conf\""
+ "(current dir).  Config file describes a prototype of library functions to be able"
```

Comment:
Ok, let's use install dir, it should also fix the problem with drltrace test failure on Appveyor. Done.